tensorboard/BUILD

```diff
@@ -96,6 +96,24 @@ py_test(
     ],
 )
 
+py_test(
+    name = "manager_e2e_test",
+    size = "large",  # spawns subprocesses, sleeps, makes requests to localhost
+    timeout = "short",  # about 15 seconds on my machine
+    flaky = True,  # on Python 2, fails about 0.5% of the time
```

Comment:
What is the cause? a bug capturing context would be ideal but a comment would suffice.

Reply:
It’s just a timeout (TensorBoard doesn’t start within the expected
period of 10 seconds). Worth noting that I’ve only seen this happening
when running with factor-12 parallelism, so it’s plausible that the
machine really was under enough load and/or IO thrashing as to not be
able to fulfill in time.

I’ll see if I can trigger it again and add a comment.